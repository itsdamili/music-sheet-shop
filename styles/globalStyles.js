// styles/globalStyles.js
import { css } from '@emotion/react';

export const globalStyles = css`
  * {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }

  body {
    font-family: 'Helvetica Neue', Arial, sans-serif;
    background-color: #fffff5;
    color: #1c1c1b;
    padding: 20px;
    line-height: 1.6;
  }

  h1 {
    text-align: center;
    margin-bottom: 20px;
    font-size: 2.5rem;
    color: #5b166d;
  }

  .container {
    max-width: 1200px;
    margin: 0 auto;
    padding: 20px;
  }

  a {
    color: inherit;
    text-decoration: none;
  }

  nav {
    background-color: #1c1c1b;
    color: #fffff5;
    padding: 10px;
    text-align: center;
    margin-bottom: 20px;
    border-radius: 8px;
  }

  nav a {
    margin: 0 15px;
    font-weight: bold;
    font-size: 1.1rem;
  }

  nav a:hover {
    color: #ccac00;
  }
`;
