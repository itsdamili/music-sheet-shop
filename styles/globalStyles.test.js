import React from 'react';
import { render } from '@testing-library/react';
import { Global } from '@emotion/react';
import { globalStyles } from '../styles/globalStyles';

describe('Global Styles', () => {
  it('should apply global styles correctly', () => {
    const { container } = render(<Global styles={globalStyles} />);
    expect(container).toMatchSnapshot();
  });
});
