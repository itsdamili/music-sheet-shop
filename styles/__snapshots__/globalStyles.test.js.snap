// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Styles should apply global styles correctly 1`] = `<div />`;
